core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -625,6 +625,8 @@ workflows:
     platform: vsphere
   openshift-psap-e2e-aws-gpu:
     platform: aws
+  openshift-e2e-external-aws:
+    platform: aws
```

Comment:
should we change this for `external` or something like `external-aws`?

Reply:
i think this just ensures that it lands on the build cluster